Telstar Logistics Parcel Delivery Solution/Data/ApplicationDbContext.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Telstar_Logistics_Parcel_Delivery_Solution.Models;
+
+namespace Telstar_Logistics_Parcel_Delivery_Solution.Data
+{
+    public class ApplicationDbContext:DbContext
+    {
+        public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options) : base(options) { }
+        public DbSet<Account> Accounts { get; set; }
+        public DbSet<Category> Category { get; set; }
+        public DbSet<City> CITY { get; set; }
```

Comment:
Does it matter that "city" is all-caps?

Reply:
ive matched the names with the Telstar Database tables
![image](https://user-images.githubusercontent.com/122527183/212161655-dca0c69b-5bfa-4874-8cfa-1dccf5c6b0dd.png)

Reply:
Ok, then we have a system in place. Great! 👍